.github/dependabot.yml

```diff
@@ -9,3 +9,6 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    ignored_updates:
+      - match:
+          dependency_name: "sigs.k8s.io/controller-runtime"
```

Comment:
I was thinking more of
```
@dependabot ignore this minor version
```
or it's analogy in config.

So in semver of MAJOR.MINOR.PATCH - the patch part would be still updated

Reply:
maybe just comment `@dependabot ignore this minor version` for now? and let it send updates in future? The evaluation of such PRs is. a minimal burden, still we will not miss the Patch upgrade if it will be released